cp3pt0-deployment/setup_tenant.sh

```diff
@@ -171,7 +185,7 @@ EOF
 )
     create_operator_group "common-service" "$OPERATOR_NS" "$target"
     install_nss
-    authorize_nss
+    #authorize_nss #authorize_nss should be done by a different command with cluster admin
```

Comment:
Why should NSS authorization be done with a different command? This will increase chances of a user forgetting to authorize a tenant namespace, and then we will run into some error during run time, which can be non-obvious to troubleshoot.

We at the very least must authorize the services-namespace in this script, otherwise the CS CR configuration step will fail. CS CR cannot have a services-namespace which cs-operator does not have access to